### ISSUE TITLE
PR self-tests fail in docker stage: `mbuffer (no such package)`

### ISSUE THREAD

**Issue by jimklimov** [oetiker/znapzend#589]
e.g. https://github.com/oetiker/znapzend/actions/runs/3297260330/jobs/5437862941

````
...
(33/33) Installing zfs (0.8.3-r1)
Executing busybox-1.31.1-r11.trigger
Executing ca-certificates-20191127-r2.trigger
OK: 67 MiB in 47 packages
fetch http://dl-3.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz
WARNING: Ignoring http://dl-3.alpinelinux.org/alpine/edge/community/: UNTRUSTED signature
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz
ERROR: unable to select packages:
  mbuffer (no such package):
    required by: world[mbuffer]
The command '/bin/sh -c apk add --no-cache zfs curl bash autoconf automake nano perl=${PERL_VERSION} openssh &&   apk add --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/community/ mbuffer &&   ln -s /dev/stdout /var/log/syslog &&   ln -s /usr/bin/perl /usr/local/bin/perl' returned a non-zero code: 1
Error: Process completed with exit code 1.
````

**Comment by jimklimov** [comment]
Empirically, this seems to only run for the first build in a PR. If I later iterate and add commits, only the 3 perl version tests run and result is green :)

**Comment by jimklimov** [comment]
So far failed to fix but can reproduce: with Alpine 3.11 it seems the trusted keys have expired vs. "community" repo (with mbuffer). Whether I use newer alpine 3.16 for newer key trust, or ignore security issues, it fails to install mbuffer because:
````
...
#6 4.130 OK: 70 MiB in 50 packages
#6 4.187 /lib/libcrypto.so.1.1
#6 4.296 /lib/libcrypto.so.3
#6 4.296 /usr/lib/libcrypto.so.1.1
#6 4.296 /usr/lib/libcrypto.so.3
#6 4.300 fetch http://dl-cdn.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz
#6 4.690 fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/main/x86_64/APKINDEX.tar.gz
#6 4.915 fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/community/x86_64/APKINDEX.tar.gz
#6 5.195 ERROR: unable to select packages:
#6 5.243   so:libcrypto.so.3 (no such package):
#6 5.243     required by: mbuffer-20220418-r1[so:libcrypto.so.3]
------
executor failed running [/bin/sh -c \
apk add --no-cache zfs curl bash autoconf automake nano perl=${PERL_VERSION} openssh libcrypto3 \
&&   apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/community/ libcrypto3 \
&&   find / -name 'libcrypto*.so*' \
&&   apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/community/ mbuffer \
&&   ln -s /dev/stdout /var/log/syslog '
&&   ln -s /usr/bin/perl /usr/local/bin/perl
]: exit code: 2
````
(list of files on top of log is from added debug `find`)

So the lib is there, dependency is not resolved, and many googlable issues over the years regarding those keywords :\

**Comment by jimklimov** [comment]
Seems like best reliable way forward is to install Alpine from main distro, and build/install custom mbuffer :\
